### PR TITLE
Lucasbranch

### PR DESCRIPTION
**Pull request by l-laurencich**
I made the homepage prettier

### CODE REVIEW

**Review by MarcelSF — CHANGES_REQUESTED** [review]
Looks good, but, should be H1 instead of H2

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,3 +1,5 @@
-<h2>This is my home page</h2>
-<p>Details about home page</p>
+<h2>This is Lucas' Super Cool Home Page</h2>
```
Comment (MarcelSF) [inline app/views/pages/home.html.erb]:
```suggestion
<h1>This is Lucas' Super Cool Home Page</h1> 
```

**Comment by l-laurencich** [comment]
I updated the header! Should be good to merge now!

**Review by MarcelSF — APPROVED** [review]
Looks great. I'm merging it.

**Pull request merged**